actions/FirefallClient.js

```diff
@@ -36,21 +36,24 @@ class FirefallClient {
     this.accessToken = accessToken;
   }
 
-  async completion(prompt, temperature = 0.0, model = 'gpt-4') {
+  async completion(prompt, temperature = 0.0, asJson = false, model = 'gpt-4') {
     const startTime = Date.now();
 
     try {
-      const response = await wretch(`${this.endpoint}/v1/completions`)
+      const response = await wretch(`${this.endpoint}/v2/chat/completions`)
         .headers({
           'x-gw-ims-org-id': this.org,
           'x-api-key': this.apiKey,
           Authorization: `Bearer ${this.accessToken}`,
           'Content-Type': 'application/json',
         })
         .post({
-          dialogue: {
-            question: prompt,
-          },
+          messages: [
+            {
+              role: 'system',
```

Comment:
It could be helpful to send a system message containing JSON requirements along with a standard message that includes the prompt.